### PR TITLE
Remove parallax when on ground

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title.

## Why It's Good For The Game

Expensive, and uneeded on ground

## Changelog
:cl:
code: Remove parallax when on ground
/:cl:

### CODE REVIEW

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Should work test it anyway

**Comment by BraveMole** [comment]
tested

**Review by TiviPlus — APPROVED** [review]

**Comment by TiviPlus** [comment]
Wait what happens if i tp from shipside to admin z level

**Comment by TiviPlus** [comment]
didnt work

File: code/_onclick/hud/parallax.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /datum/hud/proc/create_parallax(mob/viewmob)
 	var/mob/screenmob = viewmob || mymob
 	var/client/C = screenmob.client
-	if (!apply_parallax_pref(viewmob)) //don't want shit computers to crash when specing someone with insane parallax, so use the viewer's pref
+	if (!apply_parallax_pref(viewmob) || is_ground_level(viewmob.z)) //don't want shit computers to crash when specing someone with insane parallax, so use the viewer's pref
```
Comment (TiviPlus) [inline code/_onclick/hud/parallax.dm:5]: this shouldnt be needed inside create paralla itself imo

Comment (BraveMole) [reply code/_onclick/hud/parallax.dm:5]: yes

**Pull request merged**